android/src/app/organicmaps/widget/placepage/PlacePageController.java

```diff
@@ -104,51 +90,48 @@ public void onStateChanged(@NonNull View bottomSheet, int newState)
     @Override
     public void onSlide(@NonNull View bottomSheet, float slideOffset)
     {
-      final Lifecycle.State state = mMwmActivity.getLifecycle().getCurrentState();
-      if (!state.isAtLeast(Lifecycle.State.RESUMED))
-      {
-        Logger.e(TAG, "Called in the wrong activity state=" + state);
-        return;
-      }
       stopCustomPeekHeightAnimation();
       mDistanceToTop = bottomSheet.getTop();
-      mSlideListener.onPlacePageSlide(mDistanceToTop);
+      mViewModel.setPlacePageDistanceToTop(mDistanceToTop);
```

Comment:
Do we need to call here?
 ```PlacePageUtils.updateMapViewport(mCoordinator, mDistanceToTop, mViewportMinHeight);```
mDistanceToTop was changed

Reply:
To improve performance we only update the viewport when the place page is settled, which happens after the place page finished sliding.

If we call `PlacePageUtils.updateMapViewport(mCoordinator, mDistanceToTop, mViewportMinHeight);` here then the viewport will update as the place page moves.